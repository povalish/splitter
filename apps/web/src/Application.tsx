import { Loader } from '@ui-kit/components/loader/Loader';
import { Button } from '@ui-kit/components/button/Button';

//
//

export const Application: React.FC = () => (
  <div>
    <h1 className='text text-left text-2xl'>Hello from NextJS</h1>
    <div className='w-12 h-12 bg-slate-300 rounded-full' />
    <Loader />
    <Button>Click me!</Button>
  </div>
);
