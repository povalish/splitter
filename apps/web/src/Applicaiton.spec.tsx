import React from 'react';
import { render } from '@testing-library/react';

import { Application } from './Application';

describe('App', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<Application />);
    expect(baseElement).toMatchSnapshot();
  });

  it('should have a greeting as the title', () => {
    const { getByText } = render(<Application />);
    expect(getByText('Hello from')).toBeTruthy();
  });
});
