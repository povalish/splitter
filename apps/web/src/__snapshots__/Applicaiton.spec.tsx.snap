// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render successfully 1`] = `
<body>
  <div>
    <div>
      <h1>
        Hello from
      </h1>
      <div>
        Loading...
      </div>
    </div>
  </div>
</body>
`;
